presto-spark-base/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -14,22 +14,44 @@
 package com.facebook.presto.spark;
 
 import com.facebook.presto.Session;
+import com.facebook.presto.spark.classloader_interface.PrestoSparkNativeExecutionShuffleManager;
 import com.facebook.presto.testing.QueryRunner;
 import com.facebook.presto.tests.AbstractTestQueryFramework;
+import com.google.common.collect.ImmutableMap;
+import org.apache.spark.SparkEnv;
+import org.apache.spark.shuffle.ShuffleHandle;
+import org.apache.spark.shuffle.sort.BypassMergeSortShuffleHandle;
 import org.testng.annotations.Test;
 
+import java.util.Map;
+import java.util.Optional;
+
 import static com.facebook.presto.SystemSessionProperties.NATIVE_EXECUTION_ENABLED;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
 
 public class TestPrestoSparkNativeExecution
         extends AbstractTestQueryFramework
 {
+    private static final String SPARK_SHUFFLE_MANAGER = "spark.shuffle.manager";
+    private static final String FALLBACK_SPARK_SHUFFLE_MANAGER = "spark.fallback.shuffle.manager";
+
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
         return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner();
     }
 
+    public static Map<String, String> getNativeExecutionShuffleConfigs()
```

Comment:
This is a private method. Also, move it to the end of the class

Reply:
Done.